downstream/modules/platform/proc-set-custom-pod-timeout.adoc

```diff
@@ -0,0 +1,33 @@
+[id="proc-set-custom-pod-timeout"]
+
+= Custom pod timeouts
+
+You can set `awx_container_group_pod_pending_timeout` higher if long running jobs are being canceled. 
```

Comment:
```suggestion
A container group job in Controller is transitioned to the `running` state just before we submit the pod to the kubernetes API. It is then expected by Controller that the pod will enter the "Running" state within `awx_container_group_pod_pending_timeout` seconds. You can set `awx_container_group_pod_pending_timeout` higher value if it is desired that Controller wait for longer before cancelling jobs that fail to enter into the "Running" state.
```

may want to link out to the openshift docs about pod scheduling. https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/nodes/controlling-pod-placement-onto-nodes-scheduling